packages/babel-plugin-makepot/src/index.js

```diff
@@ -252,7 +252,7 @@ module.exports = function() {
 
 				// Create empty msgstr or array of empty msgstr by nplurals
 				if ( translation.msgid_plural ) {
-					translation.msgstr = Array.from( Array( nplurals ) ).map( () => '' );
+					translation.msgstr = Array.from( Array( parseInt( nplurals, 10 ) ) ).map( () => '' );
```

Comment:
Rather than parse it each time, I think it ought to be done only once in the initialization of the `nplurals` variable a few lines above in the `nplurals = pluralsMatch[ 1 ];` line.

Reply:
Oh, good point, thanks, I'll do that.